spec/acceptance/should_create_task_spec.rb

```diff
@@ -251,4 +251,56 @@
       expect(result.stdout).to match(%r{Last\sTUE})
     end
   end
+
+  it 'creates a task with synchronisation disabled: taskscheduler_api2', tier_high: true do
+    pp = <<-MANIFEST
+    scheduled_task {'#{taskname}':
+      ensure      => present,
+      compatibility => 1,
+      command     => 'c:\\\\windows\\\\system32\\\\notepad.exe',
+      arguments   => "foo bar baz",
+      working_dir => 'c:\\\\windows',
+      trigger => {
+        schedule            => daily,
+        start_time          => '12:00',
+        disable_synchronise => true,
+      },
+      provider    => 'taskscheduler_api2'
+    }
+    MANIFEST
+    apply_manifest(pp, catch_failures: true)
+
+    # Ensure it's idempotent
+    # apply_manifest(pp, catch_changes: true)
+
+    # Verify the task exists
+    query_cmd = "schtasks.exe /query /v /fo list /tn #{taskname}"
+    run_shell(query_cmd)
```

Comment:
I know this is how we verify using existing tests, but if the output that comes back from this query gives us any indicator that the `Synchronize across time zones` parameter is `on|off`, `true|false`, `0|1`, etc... it would be good to verify that

Reply:
It doesn't. I checked and it always gives the local time regardless if it's UTC or not.

Reply:
No worries - just thought I'd check.